test/test_storage_vectors.py

```diff
@@ -48,6 +50,7 @@ class TestStorageVectors(OpenWPMTest):
     def get_config(self, data_dir=""):
         return self.get_test_config(data_dir)
 
+    @pytest.mark.skip('Flash cookies are no longer supported in OpenWPM')
```

Comment:
Can I suggest xfail. "Flash cookies do not work with headless mode."

Reply:
Is it that flash cookies are no longer supported - or just no longer support with headless mode?

I was under the impression it was for both head and headless cases.

That aside - I've swapped out the skip with xfail.

Reply:
I suppose a user could still enable them if wanted. I'll make a follow-up PR that totally removes flash cookie support.